### PR TITLE
Typos and phrasing

### PR DESCRIPTION
**Pull request by fiscoal**

### CODE REVIEW

File: qna/DE-action-start-de.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 ## Question
-Which start times of an action must be printed on the receipt in germany?
+Which start times of a transaction must be printed on the receipt in Germany?
 
 ## Metadata tags
 lang-en, market-de, middleware
 
 ## Answer
-On the receipt 3 datetimes, in special cases even 4 datetimes must be printed:
+The receipt must contain three date/time pairs, in cases of long lasting transactions even four:
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:14]:
```suggestion
The receipt must contain three date/time pairs, in cases of long-lasting sales preparation processes even four:
```

File: qna/DE-action-start-de.md [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+The time of the receipt creation can be transmitted in the receipt request by using the field `ReceiptRequest.cbReceiptMoment`. It must be provided in UTC.
 
-For a receipt request of type `pos-receipt`, the ftMiddleware returns 3 dates in the signature block that can be used for printing on the receipt:
+For a receipt request of type `pos-receipt`, fiskaltrust.Middleware returns three dates in the signature block that can be used for printing the receipt:
 
-1. time of the start transaction submitted to the TSE (`ftSignatureType`: `0x4445000000000019`).
-2. time of the finish transaction submitted to the TSE (`ftSignatureType`: `0x444500000000001A`).
-3. calculated start time of the action = the minimum of the datetimes (`ReceiptRequest.cbReceiptMoment,ftChargeItem[i].Moment, ftPayItem[i].Moment`) transmitted to the ft.Middleware in the pos-receipt request. The entry has the signature type: `ftSignatureType`: `0x444500000000001F`.
+1. Time of the start transaction submitted to the TSE (`ftSignatureType`: `0x4445000000000019`).
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:25]:
```suggestion
1. Time of the start transaction submitted from the TSE (`ftSignatureType`: `0x4445000000000019`).
```

File: qna/DE-action-start-de.md [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+2. Time of the finish transaction submitted to the TSE (`ftSignatureType`: `0x444500000000001A`).
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:26]:
```suggestion
2. Time of the finish transaction submitted from the TSE (`ftSignatureType`: `0x444500000000001A`).
```

File: qna/DE-action-start-de.md [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
+3. Calculated start time of the action = the minimum of the date/time pairs (`ReceiptRequest.cbReceiptMoment,ftChargeItem[i].Moment, ftPayItem[i].Moment`) sent  to fiskaltrust.Middleware in the pos-receipt request. The entry has the signature type: `ftSignatureType`: `0x444500000000001F`.
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:27]:
```suggestion
3. Calculated start time of the Start date and time of the first transaction (order) = the minimum of the date/time pairs (`ReceiptRequest.cbReceiptMoment,ftChargeItem[i].Moment, ftPayItem[i].Moment`) sent to fiskaltrust.Middleware in the pos-receipt request. The entry has the signature type: `ftSignatureType`: `0x444500000000001F`.
```

File: qna/DE-action-start-de.md [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
 
-Depending on your usecase you can decide which returned datetimes you want to print. For example, in the normal case (no long lasting action) you may print the calculated start time of the action  (`ftSignatureType`: `0x444500000000001F`) as the start time of the action (DE: Zeitpunkt des Vorgangbeginns) on your receipt.
+Depending on your use case you can decide which of the returned times you want to print. For example, in normal cases (no long lasting action) you may print the calculated start time of the action  (`ftSignatureType`: `0x444500000000001F`) as the start time of the action (DE: Zeitpunkt des Vorgangbeginns) on your receipt.
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:29]:
```suggestion
Depending on your use case you have to decide which of the returned date and times you have to print. For example, in normal cases (no long-lasting sales preparation process) you may print the calculated start time of the transaction  (`ftSignatureType`: `0x444500000000001F`) as the start time of the transaction  (DE: Zeitpunkt des Vorgangbeginns) on your receipt.
```

File: qna/DE-action-start-de.md [hunk 6]
```diff
@@ -1,33 +1,33 @@
...
 
-If for example your usecase is a long lasting action - assuming that you transmitted the first ordered charge item having the datetime (`ftChargeItem.Moment`) of that order - you may want to use the calculated start time of the action (`ftSignatureType`: `0x444500000000001F`) as the starttime of the first order (DE: Startzeitpunkt der ersten „Bestellung“ im Bondruck.). Furthermore, you may use the returned time of the start transaction (`ftSignatureType`: `0x4445000000000019`) as the start time of the action (DE: Zeitpunkt des Vorgangbeginns).
+On the other hand, if your use case includes long lasting transactions - assuming that you sent the first ordered charge item having the datetime (`ftChargeItem.Moment`) of that order - you may want to use the calculated start time of the action (`ftSignatureType`: `0x444500000000001F`) as the start time of the first order (DE: Startzeitpunkt der ersten „Bestellung“ im Bondruck.). Furthermore, you may use the returned time of the start transaction (`ftSignatureType`: `0x4445000000000019`) as the start time of the transaction (DE: Zeitpunkt des Vorgangbeginns).
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:31]:
```suggestion
On the other hand, if your use case includes long-lasting sales preparation process - assuming that you sent the first ordered charge item having the datetime (`ftChargeItem.Moment`) of that order - you may want to use the calculated start time of the transaction (`ftSignatureType`: `0x444500000000001F`) as the start time of the first sales preparation (DE: Startzeitpunkt der ersten verkaufsvorbereitenden Vorgangs). Furthermore, you may use the returned time of the start transaction (`ftSignatureType`: `0x4445000000000019`) as the start time of the transaction (DE: Zeitpunkt des Vorgangbeginns).
```

File: qna/DE-delivery-products.md [hunk 7]
```diff
@@ -1,24 +1,24 @@
...
-With what kind of parcel service provider are you working together?
+Which courier service are you using?
 
 ## Metadata tags
 lang-en, market-de, webshop, PosDealer, PosOperator
 
 ## Answer
 
-Within a few hours after the confirmation of the order. As a general rule, products ordered UNTIL 5:30 pm will be sent on the same day.
+Within a few hours after the confirmation of the order. As a general rule, products ordered UNTIL 5:30 pm CET will be shipped on the same day.
```
Comment (AxelKutschera) [inline qna/DE-delivery-products.md:18]:
```suggestion
Products will be shipped within a few hours after the confirmation of the order. As a general rule, products ordered UNTIL 5:30 pm CET will be shipped on the same day.
```

File: qna/DE-delivery-products.md [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
 
-We are delivering from Monday to Friday.
+We are shipping from Monday to Friday.
 
-That depends on the order. We try to package/deliver as efficient as possible.
+That depends on the order. We try to package/deliver as efficiently as possible.
```
Comment (AxelKutschera) [inline qna/DE-delivery-products.md:22]:
```suggestion
The courier service depends on the order. We try to package/deliver as efficiently as possible.
```

File: qna/DE-special-VAT-cases.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ## Question
-How can I set a special case VAT ID to my charge item for the german market?
+How can I set a special case VAT ID to my charge item for the German market?
```
Comment (AxelKutschera) [inline qna/DE-special-VAT-cases.md:2]:
```suggestion
How can I set a special case VAT ID (DE: Umsatzsteuer-Identifikationsnummer/USt-IdNr.) to my charge item for the German market?
```

File: qna/DE-special-VAT-cases.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
 
 ## Question
 How can I use the iPOS interface to address special DSFinV-k VAT IDs grater than 1000?
```
Comment (AxelKutschera) [inline qna/DE-special-VAT-cases.md:5]:
```suggestion
How can I use the iPOS interface to address special DSFinV-k VAT (USt-IdNr.) IDs grater than 1000?
```

File: qna/FR-add-cashbox-cloud-en.md [hunk 11]
```diff
@@ -14,7 +14,7 @@ To create a cloud CashBox, certain conditions must be met beforehand:
 
 Once these two conditions are met, a cloud CashBox can be created in following steps:
 
-**Step One:**<br />Get in cintact with the French sales team to receive a quote for a ChaîneCloud. The quote will be free of charge, but is necessary for producing the product.
+**Step One:**<br />Get in contact with the French sales team to receive a quote for a ChaîneCloud. The quote will be free of charge, but is necessary for producing the product.
```
Comment (AxelKutschera) [inline qna/FR-add-cashbox-cloud-en.md:17]:
```suggestion
**Step One:**<br />Get in contact with the [French sales team](https://fiskaltrust.fr) to receive a quote for a ChaîneCloud. The quote will be free of charge, but is necessary for producing the product.
```

File: qna/FR-add-outlet-en.md [hunk 12]
```diff
@@ -13,7 +13,7 @@ You can now modify or correct the “SIREN” field. You must enter your SIREN n
 
 The second step is to click on the “Outlets” command in the company menu. For ease of access, you can follow the links below for the sandbox: https://portal-sandbox.fiskaltrust.fr/AccountOutlet or for the live system: https://portal.fiskaltrust.fr/AccountOutlet.
```
Comment (AxelKutschera) [inline qna/FR-add-outlet-en.md:14]:
```suggestion
The second step is to click on the “Outlets” command in the company menu. For ease of access, you can follow the links below for the sandbox: [https://portal-sandbox.fiskaltrust.fr/AccountOutlet](https://portal-sandbox.fiskaltrust.fr/AccountOutlet) or for the live system: [https://portal.fiskaltrust.fr/AccountOutlet](https://portal.fiskaltrust.fr/AccountOutlet).
```

File: qna/FR-add-outlet-en.md [hunk 13]
```diff
@@ -13,7 +13,7 @@ You can now modify or correct the “SIREN” field. You must enter your SIREN n
...
+Next, you must verify the outlet that the system already created is correct. After verification, you can add a new outlet by using the button “Add new outlet”, located on the top right side of the screen. As for an existing outlet, in order to create a new outlet, you must enter the SIRET for the outlet in the data field Location ID and the address of the outlet. The outlet number is up counting and should be the same as used in the SIRET (digits 10 to 13). The outlet number must be unique.
 
 In the fiskaltrust.Portal, you must use the numbers provided by the French authorities. The SIREN number, to be entered in the master data, is composed of nine digits. The SIRET number, used to create an outlet, corresponds to the SIREN number followed by 5 unique digits to identify the establishment. In “sandbox” mode, you can use any 5 random digits to extend the SIREN number and form a SIRET number.
```
Comment (AxelKutschera) [inline qna/FR-add-outlet-en.md:18]:
```suggestion
In the fiskaltrust.Portal, you must use the numbers provided by the French authorities. The SIREN number, to be entered in the [master data](https://portal.fiskaltrust.fr/AccountProfile/Edit), is composed of nine digits. The SIRET number, used to create an outlet, corresponds to the SIREN number followed by 5 unique digits to identify the establishment. In “sandbox” mode, you can use any 5 random digits to extend the SIREN number and form a SIRET number.
```

**Review by AxelKutschera — APPROVED** [review]
Feel free to disuss my suggestions.

File: qna/DE-action-start-de.md (hunk 2 above)
Comment (fiscoal) [reply qna/DE-action-start-de.md:25]: From? Is that value coming **from** the TSE? Probably would use _by_ then.

File: qna/DE-action-start-de.md (hunk 3 above)
Comment (fiscoal) [reply qna/DE-action-start-de.md:26]: From? Is that value coming **from** the TSE? Probably would use _by_ then.

Comment (AxelKutschera) [reply qna/DE-action-start-de.md:26]: I think these values are coming from the tse. The "by" was my 2nd option.

Comment (fiscoal) [reply qna/DE-action-start-de.md:26]: @TSchmiedlechner, are these values submitted to the TSE or do they come back from it?

File: qna/AT-carefree-extension-after-expired-grace-period.md [hunk 14]
```diff
@@ -8,7 +8,7 @@ lang-en, market-at, bundle-carefree, portal, PosOperator, PosDealer, Consultant
 
 ## Answer
 
-The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-months-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via portal, and a request for assistance will have to be placed with the fiskaltrust.Support. 
+The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-month-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via the portal and a request for assistance will have to be placed with fiskaltrust support.
```
[the remark this answers is not in the document]
Comment (magdignomad) [reply qna/AT-carefree-extension-after-expired-grace-period.md:11]: subscription via the fiskaltrust.Portal and a request for assistance will have to be placed with the PosDealer to national fiskaltrust.CustomerExperienceTeam.

File: qna/AT-carefree-extension-after-expired-grace-period.md [hunk 15]
```diff
@@ -8,7 +8,7 @@ lang-en, market-at, bundle-carefree, portal, PosOperator, PosDealer, Consultant
...
 
 **What will happen next:**
-After receiving the request for assistance, the fiskaltrust Support will manually update the expiration details of the subscription for a limited time, thus allowing the Customer to process the extension of the subscription in the usual way via the fiskaltrust.Portal. Upon the successful extension, the fiskaltrust.Support will update the subscription details accordingly to match the original purchase and expiration dates.
+After receiving the request for assistance, fiskaltrust support will manually update the expiration details of the subscription for a limited time, thus allowing the customer to process the extension of the subscription in the usual way via the fiskaltrust.Portal. Upon the successful extension, fiskaltrust support will update the subscription details accordingly to match the original purchase and expiration dates.
```
[the remark this answers is not in the document]
Comment (magdignomad) [reply qna/AT-carefree-extension-after-expired-grace-period.md:14]: fiskaltrust.CustomerExperienceTeam

File: qna/DE-delivery-products.md (hunk 8 above)
Comment (fiscoal) [reply qna/DE-delivery-products.md:22]: So it's not always UPS? In that case we should remove the last sentence.

File: qna/DE-action-start-de.md [hunk 16]
```diff
@@ -1,33 +1,33 @@
...
 ## Answer
-On the receipt 3 datetimes, in special cases even 4 datetimes must be printed:
+The receipt must contain three date/time pairs, in cases of long-lasting sales preparation processes even four:
 
-1. Date of receipt issuance/creation (DE: Datum der Belegausgabe)
-2. Start time of the action (DE: Zeitpunkt des Vorgangbeginns)
-3. End time of the action (DE: Zeitpunkt der Vorgangsbeendigung)
-4. In the special case of long lasting actions (DE: lang anhaltender Bestellvorgang): Start time of the first order (DE: Startzeitpunkt der ersten „Bestellung“ im Bondruck) 
+1. Date and time of receipt issuance/creation (DE: Datum der Belegausstellung)
+2. Start date and time of the transaction (DE: Zeitpunkt des Vorgangbeginns)
```
Comment (AxelKutschera) [inline qna/DE-action-start-de.md:17]:
```suggestion
2. Date and time of the start transaction (DE: Zeitpunkt des Vorgangbeginns)
```

**Review by AxelKutschera — COMMENTED** [review]
Typo "Start"

File: qna/AT-carefree-extension-after-expired-grace-period.md [hunk 17]
```diff
@@ -8,7 +8,7 @@ lang-en, market-at, bundle-carefree, portal, PosOperator, PosDealer, Consultant
...
 
 **What will happen next:**
-After receiving the request for assistance, fiskaltrust support will manually update the expiration details of the subscription for a limited time, thus allowing the customer to process the extension of the subscription in the usual way via the fiskaltrust.Portal. Upon the successful extension, fiskaltrust support will update the subscription details accordingly to match the original purchase and expiration dates.
+After receiving the request for assistance from the PosDealer, fiskaltrust.Customer Experience-team will manually update the expiration details of the subscription for a limited time, thus allowing the PosDealer or PosOperator to process the extension of the subscription in the usual way via the fiskaltrust.Portal. Upon the successful extension, fiskaltrust.Customer Experience-team will update the subscription details accordingly to match the original purchase and expiration dates.
```
Comment (magdignomad) [inline qna/AT-carefree-extension-after-expired-grace-period.md:14]: fiskaltrust.CustomerExperienceTeam

File: qna/DE-special-VAT-cases.md [hunk 18]
```diff
@@ -1,5 +1,5 @@
 ## Question
-How can I set a special case VAT ID to my charge item for the German market?
+How can I set a special case VAT ID (DE: Umsatzsteuer-Identifikationsnummer/USt-IdNr.) to my charge item for the German market?
```
Comment (magdignomad) [inline qna/DE-special-VAT-cases.md:2]: Is it really a good idea to have a german translation in the english text?

File: qna/DE-special-VAT-cases.md [hunk 19]
```diff
@@ -1,5 +1,5 @@
...
 
 ## Question
 How can I use the iPOS interface to address special DSFinV-k VAT (USt-IdNr.) IDs greater than 1000?
```
Comment (magdignomad) [inline qna/DE-special-VAT-cases.md:5]: DSFinV-K

**Pull request merged**